setup.py

```diff
@@ -48,7 +48,7 @@ def package_files(directory):
     "entrypoints",
     "gitpython>=2.1.0",
     "pyyaml>=5.1",
-    "protobuf>=3.7.0",
+    "protobuf>=3.19.4",
```

Comment:
3.19.4 was released on Jan 29, 2022: https://pypi.org/project/protobuf/3.19.4/

Reply:
[3.7.0](https://pypi.org/project/protobuf/3.7.0/) was released on Mar 1, 2019.

Reply:
For example, 3.7.0 does *not* seem to work, but 3.17.1 *does*

Reply:
Empirically, I've found that 3.11.2 seems to be the minimum version for which `mlflow server` starts correctly. I'm testing out this particular version here: https://github.com/mlflow/mlflow/pull/5964

Reply:
Filed https://github.com/mlflow/mlflow/pull/5965

Reply:
thanks for the investigation!